docs/3rd_party/pdf/snippets/fpdf2/create_invoice.py

```diff
@@ -0,0 +1,42 @@
+from fpdf import FPDF
+
+
+def create_custom_invoice(title, subtitle, items, filename="invoice.pdf"):
```

Comment:
```suggestion
def create_custom_invoice(title: str, subtitle: str, items: list[tuple[str | int]], filename: str = "invoice.pdf"):
```
in case it make it more clearer on what to expect. (and in case we want to inline the idea of typing from here with the Portal examples)